.github/workflows/project.yml

```diff
@@ -0,0 +1,18 @@
+name: Add to Project
+
+on:
+  issues:
+    types:
+      - opened
+  pull_request_target:
+    types:
+      - opened
+
+jobs:
+  add_to_project:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/add-to-project@v0.0.4
+        with:
+          project-url: https://github.com/orgs/conda-incubator/projects/1
+          github-token: ${{ secrets.CEPS_PROJECT_TOKEN }}
```

Comment:
I've created a Personal Access Token for the @conda-bot account and added as a repo secret to conda-incubator/ceps.